0.10/s2i/bin/assemble

```diff
@@ -48,7 +48,10 @@ if [ -n "$NPM_MIRROR" ]; then
 fi
 
 echo "---> Building your Node application from source"
-npm install -d
+case $NPM_LOG_LEVEL in
+    silent|error|warn|http|info|verbose|silly) npm install --loglevel {$NMP_LOG_LEVEL};;
+    *)             npm install;;
```

Comment:
Well yeah because the default npm value is warning. The info is quiet verbose and is what is currently present with the -d flag